### ISSUE TITLE
Crash: Flatout 1 & 2

### ISSUE THREAD

**Issue by Laserdisc9X** [Ryzee119/LithiumX#21]
Both Flatout titles will not boot under LithiumX 0.8.1, resulting in an immediate crash on a black screen;  Both titles launch under XMBC.

I tested _both titles_ as **folder** and **XISO**, and also from my personal discs AND a downloaded copy from a certain site (V);  8 tests in total.

Additionally, disc rips aside, **neither game** will boot from actual disc under LX, when using the 'Launch DVD' option.

My configuration is Cerbios 2.2.0 beta,  LithiumX 0.8.1,  Xbox v1.1.
My friends configuration is Cerbios 2.2.0 beta, LithumX 0.8.1, Xbox v1.4.

My results above also applied to him identically.

Thank you.

**Comment by Ryzee119** [comment]
Hmm both launched ok on my end.
V1.6 with m8plus bios

I'll have to dig into it. Can you try m8plus
Also PAL or NTSC copy?

**Comment by InternalLoss** [comment]
I'm wondering if it has something to do with the Dashlauncher patches - I'm unable to launch FlatOut (extracted as files), FlatOut 2 (XISO with attacher), Halo 2 (extracted as files), or SEGA GT Online (XISO + attacher). LithiumX just locks up after selecting Launch, and this also occurs on Applications such as ConfigMagic

My Xbox is a 1.6, with EvoX M8+ and Dashlauncher - launching all of these from UnleashX works just fine

**Comment by Laserdisc9X** [comment]
We used NTSC; Sounds like it might be an issue with Cerbios then...

I might be willing to go back and test M8plus, the problem is that I'm not sure it would support my SSD properly.   Cerbios filesystem configuration (made with FatXplorer) is really screwy on other bios.    I couldn't see most of the directories I setup on the SSD with FXP in M8 before flashing to Cerbios.

InternalLoss mentioned Halo 2 as folder;  boots fine for me.    I'm not sure what Dashlauncher patches are,  I haven't heard of them before;   But sounds promising. 

As I said, they did boot on XBMC.

**Comment by Ryzee119** [comment]
My flatout games are clean redump isos extracted with extractiso. Not sure what dashlaunch patches are either

**Comment by InternalLoss** [comment]
Derp, I meant [dashloader](https://github.com/Rocky5/Xbox-Softmodding-Tool/tree/master/Extras%20Disc/Hardmod/dashloader) sorry ^^

**Comment by Laserdisc9X** [comment]
I'm not sure where to find redump stuff,  not exactly in those circle.

I ripped my actual discs with DVD2Xbox, and ACL patching was disabled;   I'm not sure if DVD2X is doing something else,  or what Dashloader is still.

For making folder/file installs (from downloads),   I used ExtractIso -x;    Like wise,  I converted my DVD2X rip folders to XISO with ExtractIso -c. 

https://github.com/XboxDev/extract-xiso

Latest build was used.

**Comment by InternalLoss** [comment]
Retesting now with https://github.com/Ryzee119/LithiumX/actions/runs/5238513804 and having no issues - @Laserdisc9X would you be able to check again with the latest build? :D

**Comment by Laserdisc9X** [comment]
> Retesting now with https://github.com/Ryzee119/LithiumX/actions/runs/5238513804 and having no issues - @Laserdisc9X would you be able to check again with the latest build? :D

I'll be able to test in a few days,  out of town at the moment.    Will report back.

**Comment by Ryzee119** [comment]
Please retest in V0.9 and let me know!

**Comment by Laserdisc9X** [comment]
I just installed 0.9 and the problem still exists on my end.

Folder, XISO mount and actual disc still black screen for NTSC Flat out 1 and 2.    Additionally, while I like the new features, LX seems really choppy now;  navigation isn't as smooth,  but that's beside the point.

@InternalLoss Have you tested 0.9 specifically yet?

Edit:  I should note my setup hasn't changed,  nor do I keep up with Cerbios.  If there's a new build, someone inform me and I'll re-do the test.

**Comment by Ryzee119** [comment]
> LX seems really choppy now; navigation isn't as smooth, but that's beside the point

Bugger I'll have to try look into that. If anything it feels smoother for me. 🤔

Crashes are still a mystery

**Comment by Laserdisc9X** [comment]
> > LX seems really choppy now; navigation isn't as smooth, but that's beside the point
> 
> Bugger I'll have to try look into that. If anything it feels smoother for me. 🤔
> 
> Crashes are still a mystery

To elaborate,  there's very noticeable hiccups when navigating.   I'm more noticeable when artwork is loading in,  but it still occurs frequently, even in sub menus.   There's just a "hiccup" or a small duration of frames that feel "paused",  if you know what I mean..  a stutter.   It doesn't feel as buttery compared to v8,  which I also re-installed to "get a feel for", since it's been a little bit since I hopped on the box. (Was taking a break)

My setup hasn't changed since last time,  probably never will aside from updating Cerbios.  Out of curiosity,  since I have a pending thread for it,  I also tried the DLC installers but they also still black screen. 

Small suggestion, but certain colors make the white font unreadable.  Perhaps allow for font color config, as well.   Cheers.

EDIT:  There is a Cerbios update,  will install and report back.

EDIT 2:  Cannot flash Cerbios 2.3.1,  XBlast OS frags my Xbox when launched from LX or XBMC.

**Comment by Ryzee119** [comment]
> Small suggestion, but certain colors make the white font unreadable. Perhaps allow for font color config, as well. Cheers.

Good suggestion. I will look into this.

Youre welcome to try this https://github.com/Ryzee119/LithiumX/actions/runs/5288655113, it seemed to have a small improvement on my end but its effectively locked at 30fps like v8 but you may be more sensitive to it than me. The pause menu had all that alpha blending in the background that was slowing it down and this file seemed a tad better in that area.

I have a feeling the flatout issue is Cerbios. But hope to hear reports from others as it could be anything

**Comment by Ryzee119** [comment]
Are you able to try this?
https://github.com/Ryzee119/LithiumX/actions/runs/5308895999

I reworked alot of the lower level launch xbe stuff. Might fix Flatout maybe but I cannot verify

**Comment by Laserdisc9X** [comment]
> Are you able to try this? https://github.com/Ryzee119/LithiumX/actions/runs/5308895999
> 
> I reworked alot of the lower level launch xbe stuff. Might fix Flatout maybe but I cannot verify


Sorry for the delay,  I wasn't notified about your post from a few days ago.    I've just installed the latest linked build and will do some testing / edit my post once it's completed.     For now,  I can say that there definitely isn't any stutter after installing this build and actually the boot up after an in-game reset combo seems smooth too.

EDIT:  Still no dice on FlatOut 1&2  however I managed to finally flash the latest Cerbios 1.3.2 UDMA5.

I really don't understand why these two titles just don't function,  I don't think it has to do with Cerbios and really I don't think it's LithiumX either.   Even in XBMC,  only Flatout 1 (via attach) will boot,  the folder does not and neither any methods for Flatout 2.     The only way i've even gotten both games to boot on LithiumX, and XMBC, is with their actual physical DVDs,  but my Mountain Dew Xbox does not have a DVD drive anymore.   All methods work under UnleashX, however.

I know billy mentioned Halo 2 before,  but it never not worked for me.   It still works now.
